packages/playground-server/test/api.spec.ts

```diff
@@ -140,6 +141,28 @@ describe("playground-server", () => {
           done();
         });
     });
+
+    it("creates an account for the second time with the same username and returns HttpStatusCode.BadRequest", done => {
+      client
+        .post("/create-account", {
+          username: "joe",
+          email: "joe@joe.com",
+          address: "0x5faddca4889ddc5791cf65446371151f29653285",
+          signature:
+            "0x586b68a3362c026cdf39c63569fffb8f9106e624ef316e0f0441ea4caef4b73b0d4626f717ec23166bf061ce6728d58bcba01cd63ef8014f696dffb07d0bd3871b"
+        })
+        .catch(({ response }) => {
+          expect(response.status).toEqual(HttpStatusCode.BadRequest);
+          expect(response.data).toEqual({
+            ok: false,
```

Comment:
come on! 👺

Reply:
@ebryn I solemnly swear that my next PR is JSONAPI compliance. :pray: